src/ape/_cli.py

```diff
@@ -94,6 +95,7 @@ def get_command(self, ctx, name):
 
 
 @click.command(cls=ApeCLI, context_settings=dict(help_option_names=["-h", "--help"]))
+@plugin_helper()
```

Comment:
@lost-theory so instead of having to import logger and the `click_logging` decorator, you can just import this `plugin_helper()` and you automatically get a class that has all the logging methods available.

It also has a helpful `.abort()` method.